### DISCUSSION TITLE
Get Layer and Change Linetype?

### DISCUSSION THREAD

**Discussion by hschmiedhofer** [tarikjabiri/dxf#58] [Q&A] [answered]
Is there a way to get a reference to an existing layer via its name and change its linetype without stringifying and parsing the entire thing?

**Comment by tarikjabiri** [comment, marked as answer]
Hi,

Yes of course there is a way, you can get the  reference to a layer if exist:

```js
const layer = dxf.layer('layer_name'); // It returns undefined if not exist.
```

just make sure that the layer names do not contain some special characters: ``< > / \ "` : ; ? * | = ``.

Regards.

**Reply by hschmiedhofer** [reply]
very cool, thanks! 👍
